card/mod/basic_formats/set/all/base.rb

```diff
@@ -6,16 +6,19 @@ def show view, args
 
   # NAME VIEWS
   view :name, closed: true, perms: :none do
-    voo.variant ? card.cardname.vary(voo.variant) : card.name
+    name_variant card.cardname
+  end
+
+  def name_variant name
+    voo.variant ? name.to_name.vary(voo.variant) : name
   end
 
   view(:key,      closed: true, perms: :none) { card.key }
   view(:linkname, closed: true, perms: :none) { card.cardname.url_key }
   view(:url,      closed: true, perms: :none) { card_url _render_linkname }
 
   view :title, closed: true, perms: :none do
-    raw_title = voo.title || card.name
-    voo.variant ? raw_title.to_name.vary(voo.variant) : raw_title
+    name_variant title_in_context(voo.title)
```

Comment:
This is a bit of a shift, in which we assume that the `title` is rendered in context, and the `name` is the full name.  I'm inclined to say it makes sense to use that as a new default (but to make contextualization variants possible, perhaps via `variants`.